var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -27,10 +27,16 @@ class PyBglibpy(PythonPackage):
     version('4.1.4', commit='e54d294460e5bdf6b9990bc10a4606b412b76d90')
 
     depends_on('py-setuptools', type=('build', 'run'))
-
     depends_on('neuron+python', type='run')
+    depends_on('py-numpy@1.8:', type='run')
+    depends_on('py-matplotlib@3.0.3:', type='run')
+    depends_on('py-cachetools', type='run')
     depends_on('py-h5py~mpi@2.3:', type='run')
-    depends_on('py-bluepy', type='run')
+
+    # bluepy
+    depends_on('py-bluepy@:2.0', type='run', when='@:4.4.20')
+    depends_on('py-bluepy@2.1:', type='run', when='@4.4.36:')
```

Comment:
iirc the upper and lower bounds are included in spack.
In that case, is it ok to use bluepy 2.0 with bglibpy up to 4.4.20? Or should it be something like 1.999?

Reply:
Yes, it's ok to use bglibpy 4.4.20 with bluepy 2.0.